### PR TITLE
Revisit README

### PR DESCRIPTION
**Pull request by atugushev**
- Add Overview section
- Update supported features
- Add examples for reader and writer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,25 +6,85 @@
...
+- [x] Sampling
+- [x] AUTH
+
+## Usages
 
-## How to
+### Consumer
 
-## Examples
+A simple consumer reads a messages from "example_topic" and prints stdout.
```
Comment (lig) [inline README.md]:
```suggestion
A simple consumer reads messages from "example_topic" and prints them to stdout.
```

File: README.md [hunk 2]
```diff
@@ -6,25 +6,85 @@
...
+
+
+if __name__ == '__main__':
+    loop = asyncio.get_event_loop()
+    loop.run_until_complete(main())
+```
+
+## More examples
+
+### One connection for subscribe and publish messages
```
Comment (lig) [inline README.md:86]:
```suggestion
### One connection for subscribing and publishing messages
```

**Review by lig — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -6,25 +6,85 @@
...
+- [x] Sampling
+- [x] AUTH
+
+## Usages
 
-## How to
+### Consumer
 
-## Examples
+A simple consumer reads a messages from "example_topic" and prints them to the stdout.
```
Comment (atugushev) [inline README.md:35]:
```suggestion
A simple consumer reads messages from "example_topic" and prints them to stdout.
```

File: README.md (hunk 1 above)
Comment (atugushev) [reply README.md]: Thanks, addressed in a96222a

File: README.md (hunk 3 above)
Comment (lig) [reply README.md]: 👍🏻

**Comment by lig** [comment]
The readme definitely needed an improvement.
👍🏻

**Pull request merged**